src/obfl-specification.html

```diff
@@ -1609,48 +1609,31 @@ <h4 id="L1105" class="include">toc-entry</h4>
 <h4 id="L1105" class="include">toc-entry-on-resumed</h4>
 
 <p>The toc-entry-on-resumed element defines an entry in a table of contents. It is a
-conditional container of content for volume breaks. The contents of at most one
-toc-entry-on-resumed per volume will be rendered at the start of the resumed volume's
-table of contents.
-This functionality is only available in toc-sequences with a volume-resumed or
-document-resumed range. When there are several volume breaks in the range of a
-toc-entry-on-resumed, its contents can be rendered more than once.
-The range consists of two block ids (the latter is optional) in a notation similar to  
-<a href="https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402">Maven
-version ranges</a> that refer to the start of a block. It is required that:
-<ul>
-<li>
-   Ranges become smaller when you descend in TOC tree. More formally, if there are
-   toc-entry-on-resumed elements A and B such that the toc-block of A
-   is an <a href="https://www.w3.org/TR/2017/REC-xpath-31-20170321/#prod-xpath31-ReverseAxis">ancestor</a>
-   of the toc-block of B, then every block in B's range must be in A's range.
-</li>
-<li>
-   The order of the toc-entry-on-resumed elements is consistent with the
-   blocks they reference.
-</li>
-</ul>
-When a volume break occurs, the toc-entry-on-resumed is selected that is deepest in the
-TOC tree. The above requirements guarantee that there is at most one. 
-</p>
+conditional container of content for volume breaks. If a volume break occurs within
+its range, then its contents will be rendered at the start of the resumed volume's
+table of contents. If there are several volume breaks in the range of a
```

Comment:
Just say "... then its contents will be rendered". The rest is redundant or makes wrong assumptions.

Reply:
Done.